src/opensles/AudioOutputStreamOpenSLES.cpp

```diff
@@ -46,10 +46,10 @@ constexpr int SL_ANDROID_SPEAKER_5DOT1 = (SL_ANDROID_SPEAKER_QUAD
 constexpr int SL_ANDROID_SPEAKER_7DOT1 = (SL_ANDROID_SPEAKER_5DOT1 | SL_SPEAKER_SIDE_LEFT
         | SL_SPEAKER_SIDE_RIGHT);
 
-int AudioOutputStreamOpenSLES::chanCountToChanMask(int chanCount) {
```

Comment:
If you're going to update the parameter name it might be worth updating the method name to `channelCountToChannelMask` as well

Reply:
Good. Done.